### PR TITLE
[# 2032] - Deleted entity show error

### PR DESCRIPTION
**Pull request by muhsinkamil**
Issue Number: https://github.com/amplication/amplication/issues/2032

## PR Details

- Add tooltip for deleted entity and make the entity unclickable

## PR Checklist
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/VersionControl/PendingChange.scss [hunk 1]
```diff
@@ -36,4 +36,16 @@
...
+    background-color: var(--black10);
+    color: var(--black100);
+  }
+  &__deleted {
+    color: var(--black80);
+    cursor: pointer;
+  }
+  .tooltipped-ne:before {
+    border-top-color: var(--negative) !important;
+  }
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/VersionControl/PendingChange.scss:50]: please add a screenshot of the new tooltip showing the styling you suggested. 
@alexbass86 will be able to review and comment or approve

**Comment by yuval-hazaz** [comment]
@mshidlov I think this PR got closed by mistake when you deleted branch 11.4
I re-open the PR

**Comment by mshidlov** [comment]
Yes, please make sure you add the version label `0.11.4` next time this label suppose to alert us for open PR's that is targeting the release branch.

those labels are needed to be on the issue and not on the PR:
`@amplication/client`
`status: needs information`
`type: bug`
`type: style`

**Comment by mshidlov** [comment]
@yuval-hazaz @alexbass86 
I uploaded a staging version for you to review the change.

**Comment by mshidlov** [comment]
@alexbass86 just reviewed the changes and approved them.

**Comment by mshidlov** [comment]
@muhsinkamil thank you for your contribution I'm merging it to our next release.

**Pull request merged**